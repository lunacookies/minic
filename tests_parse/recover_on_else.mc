func main {
	if a {
		set foo = 1 +
	else {}
}
