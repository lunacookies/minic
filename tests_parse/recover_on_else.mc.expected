func main {
	if a {
		set foo = (1 + <missing>)
		<missing>
		{}
	}
}
tests_parse:35..36: error: missing operand
tests_parse:37..41: error: unmatched “else”
tests_parse:46..47: error: missing “}”
